src/main/java/use_case/signup/SignupInteractor.java

```diff
@@ -28,10 +28,10 @@ else if (!signupInputData.getPassword().equals(signupInputData.getRepeatPassword
             userPresenter.prepareFailView("Passwords don't match.");
         }
         else {
-            final User user = userFactory.create(signupInputData.getUsername(), signupInputData.getPassword());
+            final User user = userFactory.create(signupInputData.getUsername(), signupInputData.getPassword(), signupInputData.getUserID(), signupInputData.getBirthDate(), signupInputData.getFullName(), signupInputData.getEmail());
```

Comment:
Check the comment at the very top, we'll create an empty list here for moderating and posts attribute